scripts/bioformats2raw_fake

```diff
@@ -2,8 +2,9 @@
 set -e
 set -u
 image_name=ome_zarr_suite_bf2raw
-git clone git://github.com/glencoesoftware/bioformats2raw 1>&2
-cd bioformats2raw
+SUITE=ome-zarr-bf2raw-suite
```

Comment:
I now regret this name I came up with ;)

Reply:
btw, could you have just use `${image_name}` rather than making a new variable?

Reply:
I can change ``image_name`` to use ``-`` instead of ``_`` and I can reuse it.